code/src/components/FoodsList.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+export const FoodsList = () => {
+  const foodsList = useSelector(state => state.macros.foods)
+  
+  const totalCalories = () => {
+    let sum = 0
+    foodsList.map(food => {
+      return sum += food.kcal*1
+    })
+    return sum
+  }
```

Comment:
nice!
Also, take a look at the reduce() method
[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce)